tests/test_api_skeleton.py

```diff
@@ -201,6 +201,9 @@ def test_pypi_section_order_preserved(testing_workdir):
                  ('r-abf2', 'Artistic-2.0', 'OTHER', 'Artistic-2.0'),
                  ('r-cortools', 'Artistic License 2.0', 'OTHER', 'Artistic-2.0'),
                  ('r-udpipe', 'MPL-2.0', 'OTHER', ''),
+                 ('r-dt', 'GPL-3 | file LICENSE', 'GPL3', 'GPL-3'),
+                 ('r-zoo', 'GPL-2 | GPL-3', 'GPL3', 'GPL-2'),
```

Comment:
This is unfortunate. The license family should be `GPL2`. I think the problem is that the full license text `GPL-2 | GPL-3` gets passed to `guess_license_family` ([source](https://github.com/conda/conda-build/blob/master/conda_build/skeletons/cran.py#L1100)), and then `guess_license_family` searches for `GPL-3` prior to searching for `GPL-2` ([source](https://github.com/conda/conda-build/blob/master/conda_build/license_family.py#L85))

Reply:
Right. This behaviour was already there before. Let's see what I can do about it.

Reply:
Ok. Somebody invested some time in `guess_license_family` to come up with a specific matching order. Will just follow that.

Reply:
@dbast Could you share links to the previous discussion on the license order? I'm curious why this order was chosen.

Reply:
@jdblischak Well, I just read https://github.com/conda/conda-build/blame/master/conda_build/license_family.py and saw that several people worked on it.... and just assumed that there is some reasoning behind.... maybe there some extensiv discussion in some of the merged PRs...  Also the order is not obvious.... it is first trying to match GPL3, then GPL2 (like newest to oldest) and if that is not successful, following the order of `allowed_license_families` in https://github.com/conda/conda-build/blame/master/conda_build/license_family.py#L8

Reply:
Other option would be calling `guess_license_family` based on the matching of `get_license_file` ...  That would be a normalization as suggested in https://github.com/conda/conda-build/blob/master/conda_build/skeletons/cran.py#L1098

Reply:
@dbast OK, thanks for the explanation. I think your decision to follow the current convention for the skeletons makes sense.

Reply:
Closing as this was now done in a different way.